lib/mayaUsd/fileio/translators/translatorCurves.cpp

```diff
@@ -156,39 +156,49 @@ bool UsdMayaTranslatorCurves::Create(
 
         if (UsdGeomNurbsCurves nurbsSchema = UsdGeomNurbsCurves(prim)) {
             if (curveKnots.empty()) {
-                nurbsSchema.GetOrderAttr().Get(&curveOrder); // not animatable
-                nurbsSchema.GetKnotsAttr().Get(&curveKnots); // not animatable
+                nurbsSchema.GetOrderAttr().Get(&curveOrders); // not animatable
+                nurbsSchema.GetKnotsAttr().Get(&curveKnots);  // not animatable
                 nurbsSchema.GetRangesAttr().Get(
                     &curveRanges); // not animatable or actually used....
             }
 
+            const int curveDegree = curveOrders[curveIndex] - 1;
+            const int pointCount = curveVertexCounts[curveIndex];
+
+            // The USD NURBS curve schema (UsdGeomNurbsCurves) defines the number
+            // of knots as: # points + degree + 1.
```

Comment:
Note: the *official* USD docs are unfortunately wrong and claim the number of knots is the # segment + 2 * degree  + 1, The degree must not be multiplied by 2. I'll see if I can submit a documentation fix or at least report it in the USD interest group.